examples/tutorial/00_Setup.ipynb

```diff
@@ -51,9 +51,21 @@
     "    conda activate project\n",
     "    ```\n",
     "\n",
-    "4. **Download and navigate to the tutorial project**: If you've previously run the `anaconda-project download` command below, you may need to delete or rename the pre-existing `holoviz_tutorial` directory.\n",
+    "4. **Download and navigate to the tutorial project**: Download the tutorial ZIP archive from https://assets.holoviz.org/holoviz/tutorial/holoviz_tutorial.zip, unzip it and go to the `holoviz_tutorial` directory.\n",
+    "\n",
+    "    *Windows / Powershell*\n",
+    "\n",
+    "    ```powershell\n",
+    "    Invoke-WebRequest -Uri \"https://assets.holoviz.org/holoviz/tutorial/holoviz_tutorial.zip\" -OutFile \"holoviz_tutorial.zip\"\n",
+    "    Expand-Archive -Path \"holoviz_tutorial.zip\" -DestinationPath \"holoviz_tutorial\"\n",
+    "    cd holoviz_tutorial\n",
+    "    ```\n",
+    "\n",
+    "    *macOS and Linux*\n",
+    "\n",
     "    ```bash\n",
-    "    anaconda-project download pyviz/holoviz_tutorial\n",
+    "    curl -o holoviz_tutorial.zip \"https://assets.holoviz.org/holoviz/tutorial/holoviz_tutorial.zip\"\n",
+    "    unzip holoviz_tutorial.zip -d holoviz_tutorial\n",
```

Comment:
Used ChatGPT to generate these install instructions so I believe they're generic enough.